### DISCUSSION TITLE
Maveniverse project would like to join

### DISCUSSION THREAD

**Discussion by cstamas** [commonhaus/foundation#176] [Joining Commonhaus]
### Project information

- **Maveniverse, the missing pieces from Maven universe**:
- **Project site: maveniverse.eu (domain owned but no site yet, redirects to GH org site)**:
- **Code repository: https://github.com/maveniverse/**:
- **License: Eclipse Public License 2..0**:

### Do you have authority to represent this project?

- [X] I own or maintain this project

### Describe your project

While Apache Maven is hosted and developed on ASF, a small team of core Maven developers run "Maveniverse" as sort of "skunk works" for Maven to innovate, experiment and improve Maven universe, and also to enable faster paced development and contribution turnaround that that on ASF.

Maveniverse currently serves as "fast track" for several projects that may (or may not) end up in Apache Maven 'proper'. Examples of former are quick prototyping of Apache Maven Indexer Search APIs (maveniverse/mase) that was done as "fork and fix" repository, where eventually all the new features were backported to Maven indexer 'proper' (apache/maven-indexer), or the "ScopeManager" that is now part of Maven 4 'proper' (apache/maven). 

On other other hand, projects like MIMA "Mini Maven Library", Toolbox and MDK are maintained on Maveniverse.

Members of Maveniverse are all either members/committters/PMCs of Apache Maven projects, or strong contributors to same project, all of them are versed and committed open source developers.

Maveniverse, a "skunkworks" project of Apache Maven, project that had many ties with Codehaus, IMHO should really fit under umbrella that carries on Codehaus spirit.

### Things that must be done (bootstrap phase)

- [x] **Review the bylaws**, especially sections about [projects](https://www.commonhaus.org/bylaws/cf-membership.html#projects), [decision making](https://www.commonhaus.org/bylaws/decision-making.html), and [voting for significant decisions](https://www.commonhaus.org/bylaws/cf-council.html#voting).
- [x] **Read the [bootstrapping agreement](https://github.com/commonhaus/foundation/blob/main/agreements/bootstrapping/bootstrapping.md)**.
- [x] If all is well, and you agree to participate in the bootstrapping phase to help us build this thing, then **create a PR** with the following:

    1. A filled out copy of the bootstrapping agreement. You can make a copy of [bootstrapping.md](https://github.com/commonhaus/foundation/blob/main/agreements/bootstrapping/bootstrapping.md) file and check that in, or checkin a filled out [pdf](https://github.com/commonhaus/foundation/releases/download/final-draft/bootstrapping.pdf). Please follow naming conventions.
    2. An updated [CONTACTS.yaml](https://github.com/commonhaus/foundation/blob/main/CONTACTS.yaml) that names your project representative in the EGC section
    3. An updated [PROJECTS.yaml](https://github.com/commonhaus/foundation/blob/main/PROJECTS.yaml) that provides details / describes your project
    4. An updated [TRADEMARKS.md](https://github.com/commonhaus/foundation/blob/main/TRADEMARKS.md) that adds your project name and project logo (if you have one). Please include the footnote reference, as we are using your project name and logo with your permission until we have a formal agreement in place and resources are transferred.

    The commit should be "Signed-off-by" your GH login (`git commit -s -m "<project> bootstrapping agreement, yyyy-mm-dd"`)

**Comment by cstamas** [comment]
- Created https://github.com/commonhaus/foundation/pull/189

**Reply by ebullient** [reply]
🙏
